src/helper/cookie/index.test.ts

```diff
@@ -195,6 +195,118 @@ describe('Cookie Middleware', () => {
       )
     })
 
+    app.get('/get-secure-prefix-cookie', async (c) => {
+      const cookie = getCookie(c, 'delicious_cookie', { secure: true })
+      if (cookie) {
+        return c.text(cookie)
+      } else {
+        return c.notFound()
+      }
+    })
+
+    app.get('/get-host-prefix-cookie', async (c) => {
+      const cookie = getCookie(c, 'delicious_cookie', { host: true })
+      if (cookie) {
+        return c.text(cookie)
+      } else {
+        return c.notFound()
+      }
+    })
+
+    app.get('/set-secure-prefix-cookie', (c) => {
+      setCookie(c, 'delicious_cookie', 'macha', {
+        securePrefix: true,
+      })
+      return c.text('Set secure prefix cookie')
+    })
+
+    it('Set cookie with secure prefix', async () => {
+      const res = await app.request('http://localhost/set-secure-prefix-cookie')
+      expect(res.status).toBe(200)
+      const header = res.headers.get('Set-Cookie')
+      expect(header).toBe('__Secure-delicious_cookie=macha; Path=/; Secure')
+    })
+
+    it('Get cookie with secure prefix', async () => {
+      const setCookie = await app.request('http://localhost/set-secure-prefix-cookie')
+      const header = setCookie.headers.get('Set-Cookie')
+      if (!header) {
+        assert.fail('invalid header')
+      }
+      const res = await app.request('http://localhost/get-secure-prefix-cookie', {
+        headers: {
+          Cookie: header,
+        },
+      })
+      const response = await res.text()
+      expect(res.status).toBe(200)
+      expect(response).toBe('macha')
+    })
+
+    app.get('/set-host-prefix-cookie', (c) => {
+      setCookie(c, 'delicious_cookie', 'macha', {
+        hostPrefix: true,
+        domain: 'example.com', // this will be ignored
```

Comment:
```suggestion
        domain: 'example.com', // this will be ignored
        path: '/foo', // this will be ignored
        secure: false, // this will be ignored
```

Reply:
Sure!